netbox/core/models/jobs.py

```diff
@@ -216,23 +216,23 @@ def enqueue(cls, func, instance, name='', user=None, schedule_at=None, interval=
 
         return job
 
-    def trigger_webhooks(self, event):
-        from extras.models import Webhook
+    def trigger_events(self, event):
+        from extras.models import EventRule
 
         rq_queue_name = get_config().QUEUE_MAPPINGS.get('webhook', RQ_QUEUE_DEFAULT)
         rq_queue = django_rq.get_queue(rq_queue_name, is_async=False)
 
         # Fetch any webhooks matching this object type and action
-        webhooks = Webhook.objects.filter(
+        event_rules = EventRule.objects.filter(
             **{f'type_{event}': True},
             content_types=self.object_type,
             enabled=True
         )
 
-        for webhook in webhooks:
+        for event_rule in event_rules:
             rq_queue.enqueue(
-                "extras.webhooks_worker.process_webhook",
-                webhook=webhook,
+                "extras.events_worker.process_event",
```

Comment:
The handler should be determined by the action type defined on the EventRule. (`process_event` doesn't exist.)